zulipterminal/model.py

```diff
@@ -2071,3 +2071,17 @@ def poll_for_events(self) -> None:
                         self.controller.raise_exception_in_main_thread(
                             sys.exc_info(), critical=False
                         )
+
+    def show_guest_suffix(self, user_id: int) -> bool:
+        """
+        Returns True if the realm_enable_guest_user_indicator
+        setting is enabled and the user is a guest
+        """
+        user = self.get_user_info(user_id)
+        assert user is not None
+        role = user["role"]
+
+        return (
+            self.initial_data.get("realm_enable_guest_user_indicator", None)
+            and ROLE_BY_ID[role]["name"] == ROLE_BY_ID[600]["name"]
+        )
```

Comment:
I am using ( and ) condition here so if the first one returns false it will never calculate the other half of the condition right ?  
yes it looks complicated ill make it more readable

Reply:
The expression won't calculate the later part if the former is false, but `role` itself will still be calculated in the part above this, independently of that.